### PR TITLE
Add tests for question marks in contains function

### PR DESCRIPTION
**Pull request by peterstadler**
This pull request adds tests (originally from @joewiz) for issue #1379 as requested by @duncdrum.

### CODE REVIEW

**Comment by adamretter** [comment]
Thanks for your contribution @peterstadler 

Requires https://github.com/eXist-db/exist/pull/2142

**Comment by peterstadler** [comment]
> @peterstadler thank you, indeed it looks like we can close the issue when merging this PR?

sure. what's the workflow for removing the `%test:pending` annotation?

**Comment by duncdrum** [comment]
@peterstadler sry i had too many tabs open, leave it as is for now please. We'll worry about the fallout later See #2393

**Comment by adamretter** [comment]
@peterstadler #2140 is now merged. Could please rebase this on the develop HEAD? We can then look to re-examie the success/failure test of this PR.

**Comment by peterstadler** [comment]
Done.
But still I left the originally failing tests as `%test:pending`. Should I test again locally and remove this annotation?

**Comment by adamretter** [comment]
@peterstadler I think you might  have made a mistake in your rebase. This branch now contains 232 commits!

**Comment by peterstadler** [comment]
Yikes! I followed https://github.com/edx/edx-platform/wiki/How-to-Rebase-a-Pull-Request but obviously did fail. Shall I nuke this PR and create another one?

**Comment by peterstadler** [comment]
@adamretter Another try: I did a `git reset` and `git push -f` to remove the merge. Now it seems ok, the parent of the commit for this PR (690a72e8bf1d1b2344774afe08b3a8834d8aadf6) is the recent 405a82bf477a12471f4c52fda4ccb9b3cd0e69eb … I hope

**Comment by peterstadler** [comment]
@adamretter @duncdrum is there yet anything I can do here?

**Comment by dizzzz** [comment]
I restarted the Travis builds ; annoying to see appveyor is still failing for linux

File: exist-core/src/test/xquery/question-marks-in-contains-function.xql [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+declare
+        %test:pending("https://github.com/eXist-db/exist/issues/1379")
+        %test:assertEquals(1)
+function ct:parent-element-contains-question-mark() {
+    let $hits := doc("/db/test/test.xml")//tei:persName[fn:contains(., "?")]
+    return
+        count($hits)
+};
+
```
Comment (joewiz) [inline exist-core/src/test/xquery/question-marks-in-contains-function.xql]: Should this test also be marked pending?

Comment (adamretter) [reply exist-core/src/test/xquery/question-marks-in-contains-function.xql]: @peterstadler can you respond on this one please?

Comment (peterstadler) [reply exist-core/src/test/xquery/question-marks-in-contains-function.xql]: Yes, I just tested with latest 5-RC8 and indeed that test fails. (it used to work in my old v3.3.0). So I marked it as "pending" in 92ca2f0fb880be6ef1fe14a15dea551bbd3a54a8

**Comment by adamretter** [comment]
Thanks @peterstadler

**Pull request merged**